Source/Graphics/animationclient.h

```diff
@@ -89,7 +88,7 @@ class AnimationClient {
     std::string retarget_new;
 
     struct {
-        ASFunctionHandle layer_removed;
+        int32_t layer_removed;
```

Comment:
this is the same as the handle and saves us the expensive include of `ascontext.h`. It seems like the header was added lately, since ASContext already was forward declared

Reply:
Would it be possible to instead forward declare or locally define ASFunctionHandle? I feel having the typedef clarifies the intention of the code and reduces the risk of programmer mistakes.

Reply:
Yep, will change that